interp/constrexpr_ops.ml

```diff
@@ -366,6 +366,14 @@ let free_vars_of_constr_expr c =
     | c -> fold_constr_expr_with_binders (fun a l -> a::l) aux bdvars l c
   in aux [] Id.Set.empty c
 
+let free_names_of_constr_expr c =
```

Comment:
Free is misleading here, it reminds me of free variables. The comment in the mli is right, you gather both free and bound variables, but also variables standing for global constants.

All_idents_of_... Maybe?

Reply:
Just `names_of_constr_expr`?

Reply:
Indeed, this makes much more sense without "free"! Done.